src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpBuild.cs

```diff
@@ -38,10 +32,30 @@ static void Main(string[] args)
     }
 }";
 
-            _visualStudio.Instance.Editor.SetText(editorText);
+            VisualStudio.Instance.Editor.SetText(editorText);
 
             // TODO: Validate build works as expected
         }
 
+        [Fact]
```

Comment:
nit: I think we have been trying to add the appropriate traits for all the integration tests (not sure if we actually have one here though)